### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by ChrisMarriner**
Please check formatting, naming, and logic. @chanteneviljoen

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+if name in email
+    add to dictionary
+if name not in email
+    get name
+    add to dictionary
+print name and email from dictionary
+"""
+
+
+def main():
```
Comment (chanteneviljoen) [inline prac_05/emails.py:19]: Remember to add in docstrings for your functions!

File: prac_05/emails.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        if user_input.upper() != "Y" and user_input != "":
+            name = input("Name: ")
+        name_email_dict[email] = name
+        email = input("Email: ")
+    for email, name in name_email_dict.items():
+        print("{} ({})".format(name, email))
+
+
+def get_names_from_email(email):
+    prefix = email.split('@')[0]
```
Comment (chanteneviljoen) [inline prac_05/emails.py:34]: Docstring missing again

File: prac_05/state_names.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+"""
+Christopher Marriner
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
```
Comment (chanteneviljoen) [inline prac_05/state_names.py:10]: you probably don't need to print the dictionary at the start of the program?

**Comment by chanteneviljoen** [comment]
Good job, only a few minor things! 😄

File: prac_05/emails.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    add to dictionary
+if name not in email
+    get name
+    add to dictionary
+print name and email from dictionary
+"""
+
+
+def main():
+    name_email_dict = {}
```
Comment (lindsaymarkward) [inline prac_05/emails.py:20]: This is not how we suggest to name dictionaries.

**Pull request merged**